Documentation/devel/stage1-implementors-guide.md

```diff
@@ -80,6 +80,14 @@ Any stage1 that supports and expects machined registration to occur will likely
 * `--disable-paths` disables inaccessible and read-only paths (such as `/proc/sysrq-trigger`)
 * `--disable-seccomp` disables seccomp (overrides `retain-set` and `remove-set`)
 
+#### Arguments added in interface version 4
+
+* `--dns-conf-mode=resolv=(host|stage0|none|default),hosts=(host|stage0|default)`: Configures how the stage1 should set up
```

Comment:
Is there any advantage in doing it this way? It looks un-necessarily complex compared to two separate `hosts` and `resolv` flags. Also, what is the default if one (or both) are missing?